g2core/gcode_parser.cpp

```diff
@@ -779,7 +779,7 @@ static stat_t _parse_gcode_block(char *buf, char *active_comment)
             break;
 
             case 'M':
-            switch((uint8_t)value) {
+            switch((uint16_t)value) {
```

Comment:
Are you sure this is needed?

Reply:
Yes. Because, 'uint8_t' cannot process '400'.

Reply:
Ahhh, of course.  Obvious now you point it out. :smile: